src/test.rs

```diff
@@ -170,41 +125,13 @@ fn it_works_v0_parity_ed25519() {
         152, 170, 180, 218, 107, 177, 170, 58, 91, 62, 24, 240, 248, 244, 13, 51, 235, 3, 21, 63,
         79, 192, 137, 6,
     ];
-    let d = ParityDoughnutV0::decode(&mut &payload[..]).expect("It works");
+    let d = DoughnutV0::decode(&mut &payload[..]).expect("It works");
     assert_eq!(d.signature_version(), 1);
     assert_eq!(d.payload(), &payload[..payload.len() - 64]);
 }
 
 #[test]
-fn it_works_doughnut_enum_v0_parity() {
```

Comment:
The test looks to be for parity doughnut that we have maintained, couldn't we keep this?